utils/config/config.go

```
@@ -463,11 +502,28 @@ func getConfFilePath() (string, error) {
 		return "", err
 	}
 	os.MkdirAll(confPath, 0777)
-	return filepath.Join(confPath, coreutils.JfrogConfigFile), nil
+
+	versionString := ".v" + coreutils.GetConfigVersion()
+	confPath = filepath.Join(confPath, coreutils.JfrogConfigFile+versionString)
+	return confPath, nil
+}
+
+func getLegacyConfigFilePath(version string) (string, error) {
+	confPath, err := coreutils.GetJfrogHomeDir()
+	if err != nil {
+		return "", err
+	}
+	confPath = filepath.Join(confPath, coreutils.JfrogConfigFile)
+	// Before version 4 all the config files were saved with the same name.
+	if ver, _ := strconv.Atoi(version); ver < 4 {
```

Let's not ignore the error returned from ```strconv.Atoi```.